src/main/java/org/apache/sling/pipes/PipeBuilder.java

```diff
@@ -252,6 +253,16 @@
             description = "read multi property, and output each value in the bindings")
     PipeBuilder mp();
 
+    /**
+     * attach an ACL pipe to the current context
+     * @param conf configuration parameters
+     * @return updated instance of PipeBuilder
+     * @throws IllegalAccessException in case it's called with bad configuration
+     */
+    @PipeExecutor(command = "acls", resourceType = ACLPipe.RESOURCE_TYPE, pipeClass = ACLPipe.class,
+            description = "sets acls on the resource or output each acls on the resource or  acls for authorizable in repository in bindings")
+    PipeBuilder acls(Object... conf) throws IllegalAccessException;
```

Comment:
i think that the following is more appropriate as discussed:

- allow(String expr),
- deny(String expr),
- acls()

allow or deny will internally set your allow & deny flag